tests/test_htmy.py

```
@@ -138,6 +148,21 @@ def global_no_data() -> list[User]:
     async def render_component(request: Request) -> HTMLResponse:
         return HTMLResponse(await htmy.render_component(UserListItem(billy), request))
 
+    @app.get("/jinja", response_model=None)
+    @htmy.page()  # type: ignore[arg-type]
+    def jinja() -> JinjaTemplate:
+        # Verifies request injection, context processors, and empty route_params.
+        return JinjaTemplate("htmy.jinja")
+
+    @app.get("/jinja/{name}", response_model=None)
+    @htmy.page()  # type: ignore[arg-type]
+    def jinja_with_route_params(name: str) -> JinjaTemplate:
```

Ruff (0.15.20)

[warning] Unused function argument: `name`

(ARG001)